pkg/ovs/openflow/ofctrl_meter.go

```diff
@@ -127,6 +127,7 @@ func (m *meterBandBuilder) Done() Meter {
 		mbExp := new(openflow15.MeterBandExperimenter)
 		mbExp.MeterBandHeader = *m.meterBandHeader
 		mbExp.Experimenter = m.experimenter
+		mb = mbExp
```

Comment:
ditto, did it cause real issues?

Reply:
No real issue is caused by this since it is not called in current code.